cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -175,7 +175,7 @@ void Run(const std::vector<string>& bag_filenames) {
     rosbag::Bag bag;
     bag.open(bag_filename, rosbag::bagmode::Read);
     rosbag::View view(bag);
-    const ::ros::Time begin_time = view.getBeginTime();
+    const ::builtin_interfaces::msg::Time begin_time = view.getBeginTime();
```

Comment:
should this include builtin_interfaces  and remove ros/time include ? (same everywhere `ros::Time` is replaced by `builtin_interfaces::msg::Time`)